app/src/main/java/org/jellyfin/androidtv/preference/UserPreferences.kt

```diff
@@ -104,7 +105,7 @@ class UserPreferences(context: Context) : SharedPreferenceStore(
 		/**
 		 * Enable AC3
 		 */
-		var ac3Enabled = Preference.boolean("pref_bitstream_ac3", true)
+		var ac3Enabled = Preference.boolean("pref_bitstream_ac3", !DeviceUtils.isFireTvStickGen1())
```

Comment:
Do we want a migration to automatically set this (once) for all fire tv gen 1 devices?

Reply:
:+1: migration has been added.